packages/dds/tree/src/core/rebase/changeRebaser.ts

```diff
@@ -94,8 +94,8 @@ export interface ChangeRebaser<TChangeset> {
 /**
  * @internal
  */
-export interface TaggedChange<TChangeset> {
-	readonly revision: RevisionTag | undefined;
+export interface TaggedChange<TChangeset, TTag = RevisionTag | undefined> {
```

Comment:
Are we using a different tag type somewhere in this PR? What was the reason for adding this type parameter?

Reply:
Yes, the different type is `RevisionTag` as opposed to `RevisionTag | undefined`.